core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReader.java

```diff
@@ -71,9 +72,9 @@ protected TabletServerBatchReader(ClientContext context, Class<?> scopeClass, Ta
     this.tableName = tableName;
     this.numThreads = numQueryThreads;
 
-    queryThreadPool =
-        context.threadPools().getPoolBuilder("batch scanner " + batchReaderInstance + "-")
-            .numCoreThreads(numQueryThreads).build();
+    queryThreadPool = context.threadPools()
+        .getPoolBuilder(METRICS_POOL_PREFIX + "tserver.batch.reader.scanner" + batchReaderInstance)
```

Comment:
Could drop `tserver` in the name as these threads run in client code, usually outside if tserver.  Could use `client` instead to indicate a client side thread pool.

Reply:
Fixed in 21aa29c435